tcms/core/tests/test_utils.py

```diff
@@ -0,0 +1,86 @@
+import unittest
+
+from django.conf import settings
+from django.core.mail import send_mail
+from django.template.loader import render_to_string
+from unittest.mock import patch
+
+from tcms.core.utils.mailto import mailto
+
+
+class TestMailTo(unittest.TestCase):
+    @classmethod
+    def setUp(self):
```

Comment:
this is setUpClass(cls)

Reply:
Initially I started with `setUpClass` method, but after writing a couple of tests I realised that we need to update different expected parameters according to tests. Changing this to `setUpClass` method will make those parameters update for subsequent subsequent tests. Having `setUp` method ensures that consistent initial condition is maintained for each test. 

I forgot to remove the `classmethod` decorator. I think using `setUp` method is the best thing to do here.